apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -260,7 +293,7 @@ class MediaRecorderTest : public MediaRecorderObserverInterface, public enable_s
 		return input;
 	}
 
-	MediaRecorderTest() : p_out(nullptr), appRunning(false), isPaused(false) 
+	MediaRecorderTest() : p_out(nullptr), appRunning(false), isPaused(false)
 	{
 		pcmConfig.channels = 2;
```

Comment:
Remove the settings and member variables associated with the pcmConfig structure.

Reply:
Fixed